src/e-k/Fcb/Fcb.Catalog.cs

```diff
@@ -10,7 +10,7 @@ public static partial class Fcb
             .WithStyle(Style.Series)
             .WithCategory(Category.PriceChannel) // From catalog.bak.json Category: "PriceChannel"
             .AddParameter<int>("windowSpan", "Window Span", defaultValue: 2, minimum: 2, maximum: 30) // From catalog.bak.json
-            .AddResult("UpperBand", "Upper Band", ResultType.Default, isDefault: false) // From FcbResult model
+            .AddResult("UpperBand", "Upper Band", ResultType.Default, isDefault: true) // From FcbResult model - primary result
```

Comment:
no, this is still false.  there is no primary on this one

Reply:
No.  Some indicators don’t have default result values.  Default is only identifiable in result record classes that have `IReusable.Value` fields defined, where `Value` is the default.  Not all result classes are based on `IReusable` and therefore don’t have a default.  Default values are used for chainable indicators, not all are chainable.

ref: `src/_common/Reusable/IReusable.cs`

This needs to be corrected in many catalog entries